packages/svelte-ux/src/lib/utils/date.test.ts

```diff
@@ -138,32 +141,28 @@ describe('formatDate()', () => {
     for (const c of combi) {
       const [periodType, variant, locales, expected] = c;
       it(c.toString(), () => {
-        expect(formatDateWithLocale(locales, DATE, periodType, { variant, locales })).equal(
-          expected
-        );
+        expect(formatDateWithLocale(locales, DATE, periodType, { variant })).equal(expected);
       });
     }
   });
 
-  describe('should format date for PeriodType.Week', () => {
+  describe('should format date for PeriodType.Week with the good weekstarton locale', () => {
     const combi = [
-      [PeriodType.Week, 'short', defaultLocale, DayOfWeek.Sunday, '11/19 - 11/25'],
-      [PeriodType.Week, 'short', fr, DayOfWeek.Sunday, '19/11 - 25/11'],
-      [PeriodType.Week, 'long', defaultLocale, DayOfWeek.Sunday, '11/19/2023 - 11/25/2023'],
-      [PeriodType.Week, 'long', fr, DayOfWeek.Sunday, '19/11/2023 - 25/11/2023'],
-
-      [PeriodType.Week, 'short', defaultLocale, DayOfWeek.Monday, '11/20 - 11/26'],
-      [PeriodType.Week, 'short', fr, DayOfWeek.Monday, '20/11 - 26/11'],
-      [PeriodType.Week, 'long', defaultLocale, DayOfWeek.Monday, '11/20/2023 - 11/26/2023'],
-      [PeriodType.Week, 'long', fr, DayOfWeek.Monday, '20/11/2023 - 26/11/2023'],
+      [PeriodType.Week, 'short', defaultLocale, '11/19 - 11/25'],
+      [PeriodType.Week, 'short', fr, '20/11 - 26/11'],
+      [PeriodType.Week, 'long', defaultLocale, '11/19/2023 - 11/25/2023'],
+      [PeriodType.Week, 'long', fr, '20/11/2023 - 26/11/2023'],
+
+      [PeriodType.Week, 'short', defaultLocale, '11/19 - 11/25'],
+      [PeriodType.Week, 'short', fr, '20/11 - 26/11'],
+      [PeriodType.Week, 'long', defaultLocale, '11/19/2023 - 11/25/2023'],
+      [PeriodType.Week, 'long', fr, '20/11/2023 - 26/11/2023'],
```

Comment:
Updates test to show Week with automatic start of the week. 🎉

Reply:
Nice!